modules/contact-form/tracks-events.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Jetpack_Contact_Form: Add tracking for submitting contact form blocks
+ *
+ * @package    Jetpack
+ * @since      ?
+ */
+
+use Automattic\Jetpack\Tracking;
+use Automattic\Jetpack\Status;
+
+
+/**
+ * Function that sends a `jetpack_contact_form_block_message_sent` event to Tracks
+ *
+ * @param int   $post_id - the post_id for the CPT that is created.
+ * @param array $all_values - fields from the default contact form.
+ * @param array $extra_values - extra fields added to from the contact form.
+
+ * @return void
+ */
+function jetpack_tracks_record_grunion_pre_message_sent( $post_id, $all_values, $extra_values ) {
+	$status = new Automattic\Jetpack\Status();
+
+	if ( ! $status->is_development_mode() && isset( $extra_values['is_block'] ) && $extra_values['is_block'] ) {
```

Comment:
I added this check to make sure we don't send this Tracks events in development mode, but I think we should make that change more generally.

Reply:
With a quick look, I think you're right. We should add the dev mode check in this ballpark https://github.com/Automattic/jetpack/blob/a5a11059d24eb7ee50b25dbdb08f04ead55a0d50/packages/tracking/legacy/class.tracks-client.php#L54

Reply:
I'll create a PR for this

Reply:
Done here: https://github.com/Automattic/jetpack/pull/13698